firebase-app-distribution/src/test/java/com/google/firebase/appdistribution/FirebaseAppDistributionTest.java

```diff
@@ -304,6 +335,7 @@ public void updateToLatestRelease_whenSignInCancelled_checkForUpdateNotCalled()
 
   @Test
   public void updateToLatestRelease_whenSignInFailed_checkForUpdateNotCalled() {
+    Mockito.doReturn(false).when(signInStorage).getSignInStatus();
```

Comment:
nit: you can make this a static import so you don't need the Mockito prefix in front of "doReturn"